traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -2136,9 +2174,13 @@ func setNilIfEmpty(ptrs ...**string) {
 	}
 }
 
-func sanitize(ds *tc.DeliveryServiceV4) {
+func sanitize(ds *tc.DeliveryServiceV4) error {
```

Comment:
It is not possible, it seems, for this function to return a non-`nil` error - why bother returning an error value that never needs to be checked in practice?